sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -453,44 +448,21 @@ export class MessageSession extends LinkEntity {
     this.autoRenewLock =
       this.maxAutoRenewDurationInMs > 0 && this.receiveMode === ReceiveMode.peekLock;
 
+    this._isReceivingMessagesForSubscriber = false;
+    this._batchingReceiverLite = new BatchingReceiverLite(
+      context,
+      async (_abortSignal?: AbortSignalLike): Promise<MinimalReceiver> => {
+        return this._receiver!;
```

Comment:
I admittedly don't know the session cases too well so this might be obvious. Should we be attempting to initialize the receiver if the link or connection was closed for some reason and we're trying to get a batch of messages?

Reply:
Basically in sessions the general rule is "the link is created when we create the session" (ie, non-lazily) and it's never recreated afterwards.